samza-api/src/main/java/org/apache/samza/table/descriptors/RemoteTableDescriptor.java

```diff
@@ -288,7 +289,8 @@ public String getProviderFactoryClassName() {
       RateLimiter defaultRateLimiter;
       try {
         @SuppressWarnings("unchecked")
-        Class<? extends RateLimiter> clazz = (Class<? extends RateLimiter>) Class.forName(DEFAULT_RATE_LIMITER_CLASS_NAME);
+        Class<? extends RateLimiter> clazz =
+            (Class<? extends RateLimiter>) Class.forName(DEFAULT_RATE_LIMITER_CLASS_NAME);
```

Comment:
Please undo indent/superfluous changes like these, it pollutes the git blame history, also makes it harder to review things.
I believe auto-indent on your IDE may be the culprit here.

Reply:
+1